src/js/tech/html5.js

```diff
@@ -1218,7 +1218,7 @@ Html5.Events = [
   ['featuresNativeVideoTracks', 'supportsNativeVideoTracks'],
   ['featuresNativeAudioTracks', 'supportsNativeAudioTracks']
 ].forEach(function([key, fn]) {
-  defineLazyProperty(Html5.prototype, key, () => Html5[fn](), false);
+  defineLazyProperty(Html5.prototype, key, () => Html5[fn](), true);
```

Comment:
The last property here was used to see whether we should create a setter for the property or not. Given that these values do change, particularly with text tracks, we do want these to be settable but because we weren't using strict mode, it was being set without a problem. Instead, we should just make sure these are settable.